main/solution/backend/config/settings/.defaults.yml

```diff
@@ -277,3 +283,20 @@ dbTableStudyPermissions: ${self:custom.settings.dbPrefix}-DbStudyPermissions
 
 # DynamoDB table name for KeyPairs
 dbTableKeyPairs: ${self:custom.settings.dbPrefix}-DbKeyPairs
+
+# ================================ Data Egress Feature Settings ===========================================
+# NOTE: Following properties are ONLY allowed to change for the initial deployment. It's NOT recommended to change the following properties if you already have your solution deployed.
```

Comment:
Probably add if you already have egress solution enabled instead of solution deployed?

Reply:
Note updated in commit: 3b2325583b6b78128382852ff75292d7140f9fe3